src/MudBlazor.Markdown/Resources/MudBlazor.Markdown.css

```diff
@@ -34,7 +34,10 @@ pre code.hljs {
 	color: var(--mud-palette-text-secondary);
 	background-color: var(--mud-palette-drawer-background);
 	padding: 0.25em 1em;
-	margin: 0.5em 0;
+	margin: 0.5em 0 1.25em;
+}
+.mud-markdown-body blockquote p {
```

Comment:
Please add an empty line between css blocks and indentation for it as it is shown on the screenshot below.
![image](https://github.com/MyNihongo/MudBlazor.Markdown/assets/47413092/63953fbd-cc2d-41e3-a979-ef6603327c05)

Reply:
Done! :)